Library/Formula/connect.rb

```diff
@@ -0,0 +1,17 @@
+class Connect < Formula
+  desc "provides SOCKS and HTTPS proxy support to SSH."
+  homepage "https://bitbucket.org/gotoh/connect/wiki/Home"
+  url "https://bitbucket.org/gotoh/connect/get/1.104.tar.gz"
+  sha256 "6ca5291ea8e4a402d875a5507ac78d635373584fd1912f2838b93e03b99730c8"
+
+  def install
+    # gcc will produce warnings but the application will work as expected
+    system ENV.cc, "connect.c", "-o", "connect", "-lresolv"
+    bin.install "connect"
+  end
+
+  test do
+    # A poor test but a good test would require user configuration
```

Comment:
Its not an issue of config location but the details required, e.g. to test functionality we would need a SOCKS proxy + ssh server and the ability to authenticate. I will update the comment.